tests/kafkatest/tests/streams/streams_eos_test.py

```diff
@@ -159,7 +159,7 @@ def abort_streams(self, keep_alive_processor1, keep_alive_processor2, processor_
 
     def wait_for_startup(self, monitor, processor):
         self.wait_for(monitor, processor, "StateChange: REBALANCING -> RUNNING")
-        self.wait_for(monitor, processor, "processed 500 records from topic")
+        self.wait_for(monitor, processor, "processed [0-9]* records from topic")
```

Comment:
Why change here?

Reply:
The processor used here keeps track of the `numRecordsProcessed` and outputs this whenever it's a multiple of 100. It also resets in `init`, so the eos test watches for a rebalance and then looks for the `numRecordsProcessed` to hit 500 again, knowing it should have started over again from 0.
But the whole point of cooperative rebalancing is we may not have needed to revoke/reinitialize this task during a rebalance, and the counter may be reset after a rebalance.